OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - iOS crash when calling OneSignal.User.PushSubscription.Id and OneSignal.User.PushSubscription.Token when they are null.
 
+## [5.1.0]
+### Fixed
+- Add getters for the user state that includes onesignalId and externalId
```

Comment:
I think you can just put something like "Add public getters for OnesignalId and ExternalId in the User namespace" to make things simpler

I think this should be categorized under ###Changed because adding public API is a change and not a fix